### ISSUE TITLE
doc: Create a better what's next section

### ISSUE THREAD

**Issue by beraldoleal** [kytos/kytos#799]
what's next is not good. floating figures.

**Comment by hdiogenes** [comment]
https://docs.kytos.io/home/what's_next/

@beraldoleal I think we need clarification on what should be done here.

**Comment by valessiobrito** [comment]
I think it has already been done. There are two floating figures. I don't know what needs to be done.

**Comment by beraldoleal** [comment]
@valessiobrito yes, there are two floating figures but without titles or subtitles or buttons. I think that when you look into that page you have no idea what is admin or devel. We still need a better page.

**Comment by beraldoleal** [comment]
![screen shot 2018-11-06 at 14 02 05](https://user-images.githubusercontent.com/49336/48076547-8ffb7700-e1cc-11e8-8812-b52200dca5df.png)

We should add a button, label or something to identify the guides on the guy and girl.
